tests/integration/test_partition_evolution.py

```diff
@@ -100,6 +100,13 @@ def test_add_month(catalog: Catalog) -> None:
     _validate_new_partition_fields(table, 1000, 1, 1000, PartitionField(2, 1000, MonthTransform(), "month_transform"))
 
 
+@pytest.mark.integration
```

Comment:
nit: it seems like there's no test for the "default name" for any of the transforms. do you mind adding those as well? 

https://github.com/apache/iceberg-python/pull/1176/files#diff-428471774c19f5aef70af04a0b2dcbbc263c2c8b372dbc3993356e2c851539ecR310-R336

Reply:
Yes of course, I'll add them:)